_config.yml

```diff
@@ -1,14 +1,48 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
 title: PyConf Hyderabad 2019
-description: PyConf Hyderabad is an yearly tech conference about python, it's projects and development
-mail: hydpy@example.com
-twitter: hydpy
-github: https://github.com/hydpy
+mail: your-email@example.com
+description: >- # this means to ignore newlines until "baseurl:"
+  PyConf Hyderabad is an yearly tech conference about python, it's projects and development
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
 
-baseurl: '/'
+twitter: hydpy
+github:  https://github.com/hydpy
 
 tickets_link: ""
 schedule_link: ""
 
 sass:
   sass_dir: _sass
   style: compressed
+
+# Build settings
+markdown: kramdown
+plugins:
+  - jekyll-feed
+
+# Exclude from processing.
```

Comment:
Let's uncomment this and add the exclude files. `Gemfile`, `Gemfile.lock` and `vendor/` should be there for now